File: scratch/check_meta.py
import requests
from bs4 import BeautifulSoup

urls = [
    "https://www.jejutime.com/",
    "https://www.jejuqq.com/",
    "https://www.jejujapan.com/"
]

for url in urls:
    print(f"\n--- {url} ---")
    try:
        r = requests.get(url, timeout=10)
        soup = BeautifulSoup(r.text, 'html.parser')
        
        for meta in soup.find_all('meta'):
            prop = meta.get('property') or meta.get('name')
            if prop and ('og:' in prop or 'twitter:' in prop):
                print(f"{prop}: {meta.get('content')}")
    except Exception as e:
        print(f"Error: {e}")

File: scratch/check_meta.js
const https = require('https');

const urls = [
    "https://jejutime.com/",
    "https://jejuqq.com/",
    "https://jejujapan.com/"
];

async function checkUrl(url) {
    return new Promise((resolve) => {
        console.log(`\n--- ${url} ---`);
        https.get(url, (res) => {
            console.log(`Status: ${res.statusCode}`);
            if (res.headers.location) console.log(`Redirect to: ${res.headers.location}`);
            
            let data = '';
            res.on('data', (chunk) => { data += chunk; });
            res.on('end', () => {
                const head = data.split('</head>')[0];
                const metaTags = head.match(/<meta[^>]+>/g) || [];
                metaTags.forEach(tag => {
                    if (tag.includes('og:') || tag.includes('twitter:')) {
                        console.log(tag);
                    }
                });
                resolve();
            });
        }).on('error', (err) => {
            console.error(`Error: ${err.message}`);
            resolve();
        });
    });
}

async function run() {
    for (const url of urls) {
        await checkUrl(url);
    }
}

run();
